plugins/module_utils/network/dcnm/dcnm.py

```diff
@@ -152,6 +152,20 @@ def get_ip_sn_dict(inventory_data):
     return ip_sn, hn_sn
 
 
+def get_ip_sn_fabric_dict(inventory_data):
```

Comment:
Can you add a function doc string for this module to describe what it's doing.  Want to make sure we start doing this for all new functions we add as a best practice.

Reply:
Added doc string